doc/tutorials/using_windows_as_uos.rst

```diff
@@ -132,6 +143,9 @@ Pre-install drivers and re-generate Windows ISO
 
 #. Right click the downloaded ``virtio-win-0.1.141.iso`` and select ``Mount``. The ISO will be mounted to a drive;
    for example, drive ``D:``
+   OR used Oracle Driver
```

Comment:
```suggestion
   OR use Oracle Driver
```
?

Reply:
Done, already edited.

Reply:
@gvancuts Do you have any comments to add?